server/sonar-web/src/main/js/components/issue/popups/SimilarIssuesPopup.js

```diff
@@ -71,7 +71,11 @@ export default class SimilarIssuesPopup extends React.PureComponent {
           <h6>{translate('issue.filter_similar_issues')}</h6>
         </header>
 
-        <SelectList currentItem={items[0]} items={items} onSelect={this.handleSelect}>
+        <SelectList
```

Comment:
Great :+1: 
And I think the separators (like before rewrite) should be added back. It was like this : 
![image](https://user-images.githubusercontent.com/1725611/31650559-2bc9c1da-b318-11e7-97b8-3f096dab6682.png)